resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/driver/Dockerfile

```diff
@@ -17,5 +17,14 @@ ENV SPARK_HOME /opt/spark
 
 WORKDIR /opt/spark
 
-# This class will also require setting a secret via the SPARK_APP_SECRET environment variable
-CMD exec bin/spark-class org.apache.spark.deploy.rest.kubernetes.KubernetesSparkRestServer --hostname $HOSTNAME --port $SPARK_DRIVER_LAUNCHER_SERVER_PORT --secret-file $SPARK_SUBMISSION_SECRET_LOCATION
+CMD SSL_ARGS="" && \
```

Comment:
One downside here is that creators of custom images will need to stay pretty up to date with the exact version of spark-submit k8s being used for submissions.  I'd imagine that we add configuration options not infrequently.

I think it's safe to say though that docker images will be tightly tied to spark-submit versions anyway.

In general I prefer explicit over implicit so seeing this written out seems better